cmd.go

```diff
@@ -487,3 +487,69 @@ func deletecmd(opt Options) error {
 
 	return nil
 }
+
+func publishcmd(opt Options) error {
+	cmdopt := opt.Publish
+	user := nvls(cmdopt.User, EnvUser)
+	authUser := nvls(opt.Info.AuthUser, EnvAuthUser)
+	repo := nvls(cmdopt.Repo, EnvRepo)
+	token := nvls(cmdopt.Token, EnvToken)
+	tag := cmdopt.Tag
+
+	vprintln("publishing...")
+
+	if err := ValidateCredentials(user, repo, token, tag); err != nil {
+		return err
+	}
+
+	/* look up the release */
+	vprintf("%v/%v/%v: getting information for the release\n", user, repo, tag)
+	release, err := ReleaseOfTag(user, repo, tag, authUser, token)
+	if err != nil {
+		return err
+	}
+
+	vprintf("release %v has id %v\n", tag, release.Id)
+
+	/* the release create struct works for editing releases as well */
+	params := ReleaseCreate{
```

Comment:
Can you add a link to the Github docs where Github describes the types of these parameters and what they mean? Either here or on the ReleaseCreate object definition?